module/VuFind/src/VuFind/Db/Entity/AuthHash.php

```diff
@@ -38,52 +38,76 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
  *
- * @ORM\Table(name="auth_hash", uniqueConstraints={@ORM\UniqueConstraint(name="hash_type", columns={"hash", "type"})}, indexes={@ORM\Index(name="created", columns={"created"}), @ORM\Index(name="session_id", columns={"session_id"})})
+ * @ORM\Table(name="auth_hash",
+ * uniqueConstraints={@ORM\UniqueConstraint(name="hash_type",
+ *                  columns={"hash", "type"})},
+ * indexes={@ORM\Index(name="created", columns={"created"}),
+ * @ORM\Index(name="session_id", columns={"session_id"})}
+ * )
  * @ORM\Entity
  */
 class AuthHash implements EntityInterface
 {
     /**
+     * Unique ID.
+     *
      * @var int
      *
-     * @ORM\Column(name="id", type="bigint", nullable=false, options={"unsigned"=true})
      * @ORM\Id
+     * @ORM\Column(type="bigint",
```

Comment:
Another removed name -- just taking inventory of these in case we change our mind.